### PR TITLE
Update language_tr.h

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Marlin/language_tr.h [hunk 1]
```diff
@@ -59,11 +62,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Ön Isınma PLA")                                      // Ön Isınma PLA
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")                                    //
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Tüm")                                 //  Tüm
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Nozül")                               //  Nozül
```
Comment (Kaibob2) [inline Marlin/language_tr.h:65]: MSG_PREHEAT_1_END should be translated to "End" or "finished".
I guess "Nozül" means "Nozzle"

File: Marlin/language_tr.h [hunk 2]
```diff
@@ -59,11 +62,13 @@
...
 #define MSG_PREHEAT_1_BEDONLY               MSG_PREHEAT_1 _UxGT(" Tabla")                               //  Tabla
 #define MSG_PREHEAT_1_SETTINGS              MSG_PREHEAT_1 _UxGT(" Ayar")                                //  Ayar
 #define MSG_PREHEAT_2                       _UxGT("Ön Isınma ABS")                                      // Ön Isınma ABS
 #define MSG_PREHEAT_2_N                     MSG_PREHEAT_2 _UxGT(" ")                                    //
 #define MSG_PREHEAT_2_ALL                   MSG_PREHEAT_2 _UxGT(" Tüm")                                 //  Tüm
+#define MSG_PREHEAT_2_END                   MSG_PREHEAT_2 _UxGT(" Nozül")                               //  Nozül
```
Comment (Kaibob2) [inline Marlin/language_tr.h:71]: MSG_PREHEAT_2_END should be translated to "End" or "finished".
I guess "Nozül" means "Nozzle"

**Review by Kaibob2 — CHANGES_REQUESTED** [review]

File: Marlin/language_tr.h (hunk 1 above)
Comment (ghost) [reply Marlin/language_tr.h:65]: MSG_PREHEAT_1_END and MSG_PREHEAT_2_END does not end anything. When the related functions with these messages are chosen, the machine heats the hotend only. So the current translation is meaningful. There is no exact short word for Hotend in Turkish.

Comment (Kaibob2) [reply Marlin/language_tr.h:65]: Uhh, dammit, complete misunderstanding on my side. I derived it from language_en.h and interpreted MSG_PREHEAT_1_**END** as "finished" and not as "hotend".
Sorry for that.

Comment (ghost) [reply Marlin/language_tr.h:65]: No problem

**Comment by Roxy-3D** [comment]
Thank You!

**Pull request merged**